descriptors/ModuleDescriptor-template.json

```
@@ -1468,6 +1468,7 @@
         "inventory-storage.location-units.campuses.collection.get",
         "inventory-storage.location-units.institutions.item.get",
         "inventory-storage.location-units.institutions.collection.get",
+        "inventory-storage.loan-types.item.get",
```

Please add permissions for both collection and item
```suggestion
        "inventory-storage.loan-types.item.get",
        "inventory-storage.loan-types.collection.get",
```